modules/meshcloud-replicator/replicator-management-account-access/data.tf

```diff
@@ -98,7 +98,10 @@ data "aws_iam_policy_document" "meshfed_service" {
         "iam:GetRole",
         "iam:ListAttachedRolePolicies",
         "iam:ListRolePolicies",
-        "iam:GetSAMLProvider"
+        "iam:GetSAMLProvider",
+        "iam:CreateRole",
+        "iam:AttachRolePolicy",
+        "iam:UpdateAssumeRolePolicy"
```

Comment:
d: You only added the permissions mentioned in the ticket. But looking at our docs, we may need to add even more permissions? https://docs.meshcloud.io/docs/meshstack.how-to.integrate-meshplatform-aws-manually.html#set-up-aws-account-2-management

Reply:
If it works with these permissions then how can I figure out if we need more?

Reply:
Sorry, I did not see this PR anymore, as no re-request was triggered for it. I am not sure which way to go. At least adding these permissions already makes it better. But as mentioned before, perhaps we need the other permissions  that are in our public docs, too. But would also be fine for me to first go wih this set of permission